src/common/mmo.h

```diff
@@ -256,6 +256,14 @@
 #define MAX_ELESKILLTREE 3
 #endif
 
+// Maximum item options [Smokexyz]
+#ifndef MAX_ITEM_OPTIONS
+#define MAX_ITEM_OPTIONS 5
+#if MAX_ITEM_OPTIONS > 5 // (constrained to 5 through client and database !!Do not exceed!!)  [Smokexyz]
```

Comment:
`STATIC_ASSERT(MAX_ITEM_OPTIONS <= 5, "This value is limited by the client and database layout and should only be increased if you know the consequences.")`